### ISSUE TITLE
Need way to stop LHR auto-tuning, wait longer, or lower hashrate threshold to be considered LHR.

### ISSUE THREAD

**Issue by ZeddieXX** [trexminer/T-Rex#653]
The LHR auto tuning feature is great, but I think it's not waiting long enough.  Also not sure what the threshold is to be considered "LHR locked" still.

For example, I have a 3080Ti that I cannot get more than 75-76 MHs, so that's fine.  Right now I can get it at 75 MH/s at 240 W, which I'd be happy with, but t-rex would keep bumping LHR tune to 1 even if I started out at 5 (the optimum number for me).

I don't know what the threshold where t-rex still thinks it's locked (at that speed, I don't think it is, but t-rex thinks so and bumps down the LHR tune).  I think the --lhr-tune is good way to find best hashrate by trying different tunes hands-free, but there needs to be either:

1. Stop LHR auto-tuning (use a fixed tune number once you find the one you want).
2. Wait longer before deciding GPU is still in LHR mode (it takes a while for hashrate to climb).
3. Give us a way to change the hashrate threshold that's considered to be too low for an LHR unlock.

Maybe also give us a way to retain the old ordering scheme of the GPUs.  I'm still on 22.1 because I have lots of cards that I finally got the order right on the old t-rex (which was a lot of trial and error, and wished t-rex prior used the same ordering as nvidia-smi).

**Comment by Teddymeyn** [comment]
@mexarrinha have you tried multiple session of miners? like 3 GPU in one mining session and 3 in the next.

**Comment by mexarrinha** [comment]
yes, i think i finally found the problem. It has been running stable for the past hour.
I reduced all the memory clocks even tho i know that the old ones worked fine. 
In terms of hash rate it stayed the same: 
3070ti 58/61
zotacs/ichill/aorus 3060ti 42/44
Lowering the power definitely affects the hash rate immediately

**Comment by ZeddieXX** [comment]
> 
> 
> Are you able to get 6 gpu mining on windows? Mine keeps crashing and complaining from different gpus.. Without LHR runs normal no problems, In hiveOS runs fine, on windows keeps crashing. If i try only 2/3 gpus it works fine

How is this relevant to this issue?  Shouldn't this be asked in a support forum or discord?  Anyways, glad you found a fix though.

Do you know how to stop the LHR auto-tuning from tuning down automatically?  Like make it a fixed LHR value?

**Comment by mexarrinha** [comment]
No, i spent all day trying every combination. It even goes down if you set it to default overclocking settings.
Try setting power limit all the way down and CC MC default fan in auto.
Then try increasing everything. I was able to go from PL 45 to PL 80 without LHR tunning kick in.
Then i tried turning off the miner and turn it back on and it detected LHR. I think trex is messing with the gpu memory via code with kernel and stuff. My 3070ti memory clock is at 1550 if i go higher lhr kicks in until lhr 4. I always start with LHR 8. It would be good if they had auto tune plus a choice for users to choose afterwards like teamredminer does with eth config.
